extensions/adobe/experience/decisioning/proposition.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Proposition",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "description": "The  Decision Proposition XDM is used to capture the outcome of a decisioning process",
+  "definitions": {
+    "proposition-mixin": {
+      "meta:intendedToExtend": [
+        "https://ns.adobe.com/xdm/context/experienceevent",
+        "https://ns.adobe.com/experience/decisioning/decisionevent"
+      ],
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositionId": {
+          "title": "Decision Proposition ID",
+          "type": "string",
+          "description": "An ID associated with an individual decision response. A single decision can produce an array of propositions, one for each activity involved. It is the expectation that applications reference this ID in every experience event that was influended by the propositions. A user profile's proposition-response history is maintained with these identifiers."
```

Comment:
Identifier not ID in titles. and always call "experience event" ExperienceEvent

Reply:
Fixed